.vscode/settings.json

```diff
@@ -60,5 +60,5 @@
             ]
         }
     ],
-    "vorlage-latex.buildcontainer": "jemand771/latex-build:v1.1.1"
+    "vorlage-latex.buildcontainer": "jemand771/latex-build:experimental"
```

Comment:
willst du wrkl was mergen, was auf deinen experimental container verweist? das muss man quasi vergessen beim in master mergen

Reply:
Ich denke das ist erstmal nur provisorisch :D

Reply:
"nicht mergen, das docker-image steht noch auf experimental bis ich da ne neue version raus hab ^^" - sry kann nich leysen